src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,22 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File file = new File(fileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file", e);
+        }
```

Comment:
doesn't it work without creating a new file?

Reply:
nope

Reply:
I have moved in same try catch block